pixi.toml

```diff
@@ -20,14 +20,14 @@ channels = ["conda-forge"]
 # Default feature configuration
 
 [dependencies] # == [feature.default.dependencies]
-pip = "*"        # Required to install from PyPI
-jupyterlab = "*" # JupyterLab for notebooks
+pip = ">=25.2,<26"        # Required to install from PyPI
+jupyterlab = ">=4.4.7,<5" # JupyterLab for notebooks
```

Comment:
Setting the lower limit helps with installations into dirty environments.
Setting the upper limit helps with potential API breakages in new module releases, but makes us blind to updates. Maybe it is better to just use the low limit here?
If jupyterlab 5.x is available on pypi, we should get immediate response from our daily builds

Reply:
These pins were added automatically by `pixi upgrade`. Let’s revert to unpinned versions for simplicity and rely on CI to alert us if new releases introduce issues.